source/slang/slang-ast-type.h

```diff
@@ -903,25 +903,17 @@ class NamespaceType : public Type
 
 // The concrete type for a value wrapped in an existential, accessible
 // when the existential is "opened" in some context.
-FIDDLE()
-class ExtractExistentialType : public Type
+FIDDLE(abstract)
+class InterfaceWithContext : public Type
```

Comment:
The comment on this declaration is no longer accurate to whatever this type is now. The comment applies to the `ExtractExistentialType` case, but I don't think it can possibly be right for the `some` type case, since `some` types aren't existentials.